### PR TITLE
Corrections

### PR DESCRIPTION
**Pull request by aditya-rajagopal**
You were printing the validation error instead of the training error
Also  you were training on the entire dataset after finding optimal C which is not what you have to do in question b
Now the only thing I am not sure about is if you have to train on the train+validation set or only the train set
But definitely not the entire set because they only told you to report train accuracy which is only on the training set

### CODE REVIEW

**Comment by aditya-rajagopal** [comment]
Added an example output.
